piton/tools/src/proto/vivado/gen_project.tcl

```diff
@@ -81,7 +81,11 @@ add_files -norecurse -fileset $fileset_obj $files_to_add
 foreach inc_file $ALL_INCLUDE_FILES {
     if {[file exists $inc_file]} {
         set file_obj [get_files -of_objects $fileset_obj [list "$inc_file"]]
-        set_property "file_type" "Verilog Header" $file_obj
+        #if {[file extension $inc_file] == ".svh"} {
```

Comment:
Could we check the blackparrot flag for this? I am a little concerned about declaring some of the old verilog as systemverilog

Reply:
oh I don't think we actually don't need this anymore

Reply:
Reverted